IBAnimatable/PresentationKeyboardTranslation.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Created by Tom Baranes on 13/08/16.
+//  Copyright © 2016 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+public enum ModalKeyboardTranslation {
+  case None
+  case MoveUp  
+  case AboveKeyboard
+
+  static public func fromString(string: String?) -> ModalKeyboardTranslation {
+    if string == "MoveUp" {
```

Comment:
Yes, I am OK with your implementation, and I think the Swift 3 adoption will be very quick.